### PR TITLE
[http][manager] Add manager related http interface.

### PR DESCRIPTION
**Pull request by luozenglin**
## Proposed changes

Encapsulate some http interfaces for better management and maintenance of doris clusters.

The http interface includes getting cluster connection information, node information, node configuration information, batch modifying node configuration, and getting query profile.

For details, please refer to the document:  
`docs/zh-CN/administrator-guide/http-actions/fe/manager/`

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java [hunk 1]
```diff
@@ -17,6 +17,9 @@
 
 package org.apache.doris.common.profile;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java:20]: import order

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/HttpUtils.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+public class HttpUtils {
+    static final int REQUEST_SUCCESS_CODE = 0;
+
+    static List<Pair<String, Integer>> getFeList() {
+        return Catalog.getCurrentCatalog().getFrontends(null)
+                .stream().filter(Frontend::isAlive).map(fe -> new Pair<>(fe.getHost(), Config.http_port))
+                .collect(Collectors.toList());
+    }
+
+    static String concatUrl (Pair<String, Integer> ipPort, String path, Map<String, String> arguments) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/HttpUtils.java:57]:
```suggestion
    static String concatUrl(Pair<String, Integer> ipPort, String path, Map<String, String> arguments) {
```

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java [hunk 3]
```diff
@@ -0,0 +1,832 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.httpv2.rest.manager;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java:20]: import order

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java [hunk 4]
```diff
@@ -0,0 +1,832 @@
...
+@RestController
+@RequestMapping("/rest/v2/manager/node")
+public class NodeAction extends RestBaseController {
+    private static final Logger LOG = LogManager.getLogger(NodeAction.class);
+    private static final Pattern PATTERN = Pattern.compile(":");
+
+    public static final String AUTHORIZATION = "Authorization";
+    private static final int HTTP_WAIT_TIME_SECONDS = 2;
+
+    public static final String CONFIG = "配置项";
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java:82]: We should only return column name in English. The multilanguage support should be done in Manager Frontend.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/QueryProfileAction.java [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.httpv2.rest.manager;
+
+import com.google.common.base.Strings;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/QueryProfileAction.java:20]: import order

File: fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java (hunk 1 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java]: done.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/HttpUtils.java (hunk 2 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/HttpUtils.java]: done.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java (hunk 3 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java]: done.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java (hunk 4 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/NodeAction.java:85]: Maybe we can change it to English in a later version.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/QueryProfileAction.java (hunk 5 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/manager/QueryProfileAction.java]: done.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**